core/src/main/java/org/apache/struts2/interceptor/BackgroundProcess.java

```diff
@@ -86,7 +86,7 @@ protected void beforeInvocation() throws Exception {
      *
      * @throws Exception any exception thrown will be thrown, in turn, by the ExecuteAndWaitInterceptor
      */
-    protected void afterInvocation() throws Exception {
+    private void afterInvocation() throws Exception {
```

Comment:
Same here, using `protected` allows users re-implementing this class easily

Reply:
Thanks 👍 . fixed :)